src/AutorestSwift/JsonRpc/AutorestPlugin.swift

```diff
@@ -129,7 +138,11 @@ class AutorestPlugin {
     }
 
     func handleListInputs(response: RPCObject) {
-        let filename = response.asList?.first?.asString ?? ""
+        guard let filename = response.asList?.first?.asString else {
+            logger.log("handleListInputs filename is nil")
+            return
```

Comment:
We can probably just make this fatalError with the same message. If this fails, logically the whole process should fail.